src/game.py

```diff
@@ -1,46 +1,75 @@
 from board import Board
 from os import system
+from os import get_terminal_size
 
 
 class Game:
-    OBSTACLE = '# '
-    EXIT = '  '
-    PUCK = {Board.RED: 'O ', Board.BLUE: '0 ', Board.BLACK: '* '}
+    TOP_PADDING = int(get_terminal_size()[1]*0.20)
+    LEFT_PADING = int((get_terminal_size()[0]-50) / 2)
+    WHITE_BG = '\033[47m'
+    GREY_BG = '\033[47m'
+    END_COLOR = '\033[0m'
+    OBSTACLE = WHITE_BG + '# ' + END_COLOR
+    EXIT = WHITE_BG + '  ' + END_COLOR
+    EMPTY_CELL = WHITE_BG + '  ' + END_COLOR
+    PUCK = {Board.RED: WHITE_BG + '\033[31mO ' + END_COLOR,
+            Board.BLUE: WHITE_BG + '\033[34m0 ' + END_COLOR,
+            Board.BLACK: WHITE_BG + '\033[90m0 ' + END_COLOR}
 
     def __init__(self):
         self.board = Board()
-        self.pucks = {Board.RED: 6, Board.BLUE: 6}
+        self.we_have_a_winner = False
+        self.current_player = Board.RED
+        self.pucks = {Board.RED: 6, Board.BLUE: 6, Board.BLACK: 1}
```

Comment:
Can you initialize this with actual length of the initial pucks arrays ?
To ensure it is always consistent

Reply:
even better I think, I'm going to add a method on Board to get a count of the remaining puck of a certain color, this way it will always be consistent.